src/systems/filecoin_markets/storage_market/storage_market_actor.id

```diff
@@ -16,6 +17,15 @@ type StorageMarketActorState struct {
     // only the _portion_ of the total escrow amount that is locked.
     LockedReqTable  actor.BalanceTableHAMT
 
+    _rtUpdatePendingDealStatesForParty(rt Runtime, addr addr.Address)
+    _rtUpdatePendingDealStates(rt Runtime, dealIDs [deal.DealID], epoch block.ChainEpoch)
```

Comment:
The intent of `deal.DealIDs` was to wrap `[]deal.DealID` so that a serialization method would be generated for it. I don't think the wrapper is necessary for native helper function calls (though don't have a strong preference).